example_commands.md

```diff
@@ -11,7 +11,7 @@ docker run --rm --net cbio-net \
     -v "$PWD"/study-dir:/study:ro \
     -v "$HOME"/Desktop:/outdir \
     cbioportal-image \
-    metaImport.py -u http://cbioportal-container:8080/cbioportal -s /study --html=/outdir/report.html
+    sh -c 'exec metaImport.py --jar_path=/usr/local/tomcat/webapps/cbioportal/WEB-INF/lib/core-*.jar -u http://cbioportal-container:8080/cbioportal -s /study --html=/outdir/report.html'
```

Comment:
Update: it probably does, the Perl scripts `env.pl` and `envSimple.pl`, used by the various command line wrapper scripts, make the same assumption of `$PORTAL_HOME` being a directory with both source code and compiled Java.